gateway/server.go

```diff
@@ -62,7 +62,11 @@ func main() {
 
 	metricsOptions := metrics.BuildMetricsOptions()
 	exporter := metrics.NewExporter(metricsOptions, credentials)
-	exporter.StartServiceWatcher(*config.FunctionsProviderURL, metricsOptions, "func", servicePollInterval)
+	if err := exporter.StartServiceWatcher(*config.FunctionsProviderURL,
```

Comment:
This line break is a little odd, it will read better on one line _or_ with each arg on a new line. In the second case the if block should follow the method call.

Reply:
Done